x/authority/types/authorizations.go

```diff
@@ -0,0 +1,186 @@
+package types
+
+import (
+	"fmt"
+
+	"cosmossdk.io/errors"
+)
+
+// DefaultAuthorizationsList list is the list of authorizations that presently exist in the system.
+// This is the minimum set of authorizations that are required to be set when the authorization table is deployed
+func DefaultAuthorizationsList() AuthorizationList {
```

Comment:
Wondering if we couldn't rather have 3 global vars. `OperationalMsgs`, `EmergencyMsgs` and `AmindMsgs`, then `DefaultAuthorizationsList` computes the array dynamically from the 3 variables.

So looks more visual for the lists

Reply:
c18bd7d3776754598a3ebc3a527995e71faf7f15